src/Persistence/MassTransit.EntityFrameworkCoreIntegration/Configuration/EntityFrameworkOutboxConfigurationExtensions.cs

```diff
@@ -347,9 +353,21 @@ public static void ConfigureOutboxMessageEntity(this EntityTypeBuilder<OutboxMes
 
             outbox.Property(p => p.ContentType)
                 .HasMaxLength(256);
-            outbox.Property(p => p.MessageType);
 
             outbox.Property(p => p.Body);
         }
+
+        /// <summary>
+        /// Configures the entity type builder to opt out of Entity Framework conventions.
+        /// This method sets the maximum length of all properties to null, effectively removing any length constraints.
+        /// </summary>
+        /// <param name="builder">The EntityTypeBuilder instance to configure.</param>
+        static void OptOutOfEntityFrameworkConventions(this EntityTypeBuilder builder)
+        {
+            foreach (var properties in builder.Metadata.GetProperties())
```

Comment:
Some properties actually have limits, like the URLs are `HasMaxLength(256)` - I'd prefer to not change any of the existing mappings, just prevent conventions from changing the unspecified ones.

Reply:
It won't override presiding configuration, only previous configuration (including conventions). So, the MassTransit specific config will remain. As will all presiding user defined configuration.

Reply:
The advantage with this approach is that if MassTransits entities were to be extended with other unlimited sting properties, this issue is already solved for them as well. Or in other words: the config for MassTransits entities should start at the point of registration (`AddTransactionalOutboxEntities`) and be explicitly configured from that point. Thats my reasoning anyway 😅 

But I'll target the specific properties if you'd like 🙂 Just say the word.

Reply:
If the order does the trick, I'm good with it :)